JitsiConference.js

```diff
@@ -1403,10 +1426,7 @@ JitsiConference.prototype._setTrackMuteStatus = function(mediaType, localTrack,
     if (FeatureFlags.isSourceNameSignalingEnabled()) {
         // TODO When legacy signaling part is removed, remember to adjust signalingLayer.setTrackMuteStatus, so that
         // it triggers sending the presence (it only updates it for now, because the legacy code below sends).
-        this._signalingLayer.setTrackMuteStatus(
-            getSourceNameForJitsiTrack(this.myUserId(), mediaType, 0),
-            isMuted
-        );
+        this._signalingLayer.setTrackMuteStatus(localTrack?.getSourceName(), isMuted);
```

Comment:
what's the point of calling this method if `localTrack` is `undefined`?

Reply:
I will be revisiting all the presence related stuff in a another PR. The only time this will be undefined is when there are no local video tracks and that point I don't know what source name will be sent in the presence.

Reply:
Ok got it. If there's no local track then the track should be removed from the presence.